src/mesh.cpp

```diff
@@ -1619,6 +1626,23 @@ void MOABMesh::initialize() {
   build_kdtree(ehs_);
 }
 
+void
+MOABMesh::create_interface()
+{
+  // Do not create a MOAB instance if one is already in memory
+  if (!mbi_) {
```

Comment:
I'll suggest we change this line to

```cpp
if (mbi_) return;
```

And unindent the remaining code in the if statement.

Reply:
Addressed in commit ffeed77d5d4437d0dc6aadbf42843a9b565208dc